### PR TITLE
Add over the moon coordinator

### LINKED ISSUE #1586: [federation]: coordinator 'Over the Moon'
### Coordinator Alias

Over the Moon

### Avatar

otm200

### Description

Forget "to the moon", at Over the Moon we are here to take no-KYC, p2p exchange to the next level! Our goal is to further decentralise the RoboSats federation whilst providing the maximum level of privacy and security for our users. *Trade amounts increase by 30% every 2016 blocks (~2 weeks). 

### Motto

Bene videtur hinc

### Color

#eeeee4

### DevFund Donations

25

### PGP Public Key and Fingerprint

https://keys.openpgp.org/vks/v1/by-fingerprint/19D833CB2258715B4E627291942B7D51A8533742

### Privacy Policy

1. Evidence in Disputes: In the event of a dispute, users will be asked to provide transaction-related evidence. This could include transaction IDs, screenshots of payment confirmations, or other pertinent transaction records. Personal information or unrelated transaction details should be redacted to maintain privacy.
2. Short-term Storage: Sensitive information related to disputes will be stored only for the duration necessary to resolve the issue. Once resolved, the data will be permanently deleted.
3. For ease of use e-mail communications are permitted, but SIMPLEX is advised for maximum privacy.

### Data Policy

- **No Third-Party Sharing**: Under no circumstances will user information be shared with third parties. Also because it is impossible to do so by design of Robosats architecture. Due to the decentralized and privacy-centric structure of RoboSats built on the Tor network, we are unable to collect or store user information, including but not limited to, the location of users. This design ensures maximum privacy and anonymity for our users.
- **Short-term Storage**: Any log needed to operate the coordinator will be cleared after 7 days, unless strictly needed to process disputes. To ensure utmost privacy, the coordinator will be accessible only through TOR.

### Other Policies

Rule 1: Do not share personal information through the chat, unless strictly needed for completing the trade.
Rule 2: Users are solely responsible for understanding and complying with their local laws and regulations regarding cryptocurrency transactions. By using this coordinator, you agree to ensure that your activities on the platform are lawful in your jurisdiction.

### Onion Mainnet Coordinator Service Address

http://otmoonrndnrddqdlhu6b36heunmbyw3cgvadqo2oqeau3656wfv7fwad.onion

### Onion Testnet Coordinator Service Address

http://otmtestgbj3kqo3nre6oksusuqfb4ids5zg2y5z2qza2jogeu67stwid.onion

### I2P Mainnet Coordinator Service Address

_No response_

### I2P Testnet Coordinator Service Address

_No response_

### Clearnet Mainnet Coordinator Service Address

_No response_

### Clearnet Testnet Coordinator Service Address

_No response_

### Mainnet Pubkey(s)

023924542082a5d16bce188ec4c29a45f00dd439a3f5992034d82e3353232a0345

### Testnet Pubkey(s)

02f0ddc838b35fe54daa13baa4abab84475c7b9f2670ff4b53c1724792843ef62a

### 1. email

otmoon@proton.me

### 2. SimpleX

https://simplex.chat/contact#/?v=2-7&smp=smp%3A%2F%2FUkMFNAXLXeAAe0beCa4w6X_zp18PwxSaSjY17BKUGXQ%3D%40smp12.simplex.im%2FtrEIGy97lLpA9IeAc6axGF6J6vWK6U6r%23%2F%3Fv%3D1-3%26dh%3DMCowBQYDK2VuAyEAj8vhZDsAf8au1XSFG43DWZzzFYUnGNi9RAXMVvG2znk%253D%26srv%3Die42b5weq7zdkghocs3mgxdjeuycheeqqmksntj57rmejagmg4eor5yd.onion

### 3. Matrix

_No response_

### 4. Nostr

npub10tm00n7rhl0c4fja7fr94fuyzzt04rhxkt2dzn7y8kt5uhde4wtq899xam

### 5. Website

_No response_

### 6. Telegram

_No response_

### 7. Twitter

_No response_

### 8. Reddit

_No response_

### RoboSats Federation Basis

- [X] I accept the basis of the RoboSats Federation

### PR DESCRIPTION
**Pull request by KoalaSat**
## What does this PR do?
Fixes #1586

Add over the moon coordinator

## Checklist before merging
- [x] Install [pre-commit](https://pre-commit.com/#installation) and initialize it: `pip install pre-commit`, then `pre-commit install`. Pre-commit installs git hooks that automatically check the codebase. If pre-commit fails when you commit your changes, please fix the problems it points out.

### CODE REVIEW

File: frontend/static/federation.json [hunk 1]
```diff
@@ -142,6 +142,52 @@
     "mainnetNodesPubkeys": ["02c5b5972b05fba2cd2c2d9269a47bc478f73fae0f248a85cb1e5af60a07c1919d"],
     "testnetNodesPubkeys": ["032b698c8143f293d138c0926594f11d119194ddedb513f63a944d14c094d0e54a"]
   },
+  "overthemoon": {
```
Comment (otmooon) [inline frontend/static/federation.json:145]: `"overthemoon"` can be changed to `"moon"`

File: frontend/static/federation.json [hunk 2]
```diff
@@ -142,6 +142,52 @@
...
+    "longAlias": "Over the moon",
+    "shortAlias": "overthemoon",
```
Comment (otmooon) [inline frontend/static/federation.json:147]: `"shortAlias"` can be changed to `"moon"`, everything else looks good. Thank you so much!

**Review by otmooon — CHANGES_REQUESTED** [review]
Just a couple of comments shortening "overthemoon" to "moon". Everything else looks good (to me at least). Thank you so much!

**Comment by KoalaSat** [comment]
Thanks for the review! These alias you see here are just used only on the tech side so it doesn't really affects or reach to the end user

**Comment by Reckless-Satoshi** [comment]
Hey, this coordinator won't be reachable for those using the self-hosted app.

We need to add the correct nginx configs in https://github.com/RoboSats/robosats/tree/main/nodeapp/coordinators

It's not too relevant. But I also favor `moon` over `overthemoon` . Mostly because the short alias will become part of URLs and more concise urls are always nice.

**Pull request merged**